fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/ReindexManager.java

```diff
@@ -160,4 +169,38 @@ public void rollback() {
     public void shutdown() {
         ocflStream.close();
     }
+
+    private Thread startReporter() {
+        final var reporter = new Thread(() -> {
+            final var startTime = Instant.now();
```

Comment:
An log statement here (or wherever you think is appropriate) communicating to the users that they can expect progress updates every 5 minutes (or whatever the reporting interval is) so they're not left hanging for 5 minutes not knowing what's going on.